test/workbox-background-sync/node/lib/test-StorableRequest.mjs

```diff
@@ -0,0 +1,142 @@
+/*
+ Copyright 2017 Google Inc. All Rights Reserved.
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+import {expect} from 'chai';
+import sinon from 'sinon';
+import StorableRequest from
+    '../../../../packages/workbox-background-sync/lib/StorableRequest.mjs';
+
+
+describe(`StorableRequest`, function() {
```

Comment:
Please add `[workbox-background-sync]` to the top level describe call.

Reply:
Added, but I originally didn't add anything here since this object isn't publicly exposed. Do you have thoughts on how we annotate that?